README.md

```diff
@@ -1 +1,158 @@
-# Catalog
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog <a name="about-project"></a>
+
+[Catalog] is a console app designed to assist in maintaining records of various types of items such as books, music albums, movies, and games. The data is stored in JSON files, and a database has been created with tables structured analogously to the program's class hierarchy.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Ruby
+- Rspec
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Project Demonstration
+View a video demo of our project [here](https://drive.google.com/file/d/1U8LVpmkEmn5zLwWKFKxGYYIRdgCJbmuC/view?usp=sharing)
```

Comment:
- [ ] Great work on the video presentation however the video ended strangely while you were talking, kindly consider recording a new one in which all the students in the group have a chance to talk and present a piece of code you're proud of making, and please try to keep it 5 minutes max